### ISSUE TITLE
IDISP003 should not warn when serial disposable is assigned in lambda

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/IDisposableAnalyzers#251]
```cs
meh.ObserveValue(x => x.P)
   .Subscribe(x => _serial.Disposable = Create()),
```
